hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.tsx

```diff
@@ -2,22 +2,30 @@ import React from "react";
 import styles from "./TeamCard.module.scss";
 import Container from "@material-ui/core/Container";
 import Typography from "@material-ui/core/Typography";
-import Button from "@material-ui/core/Button";
 import TitledPaper from "components/general/TitledPaper/TitledPaper";
+import Button from "@material-ui/core/Button";
 
-const TeamCard = ({ members, teamCode, handleEditTeam }) => {
-    const title = teamCode === "" ? "Team" : "Team " + teamCode;
+interface TeamProps {
+    members?: string[];
+    teamCode?: string;
+    handleEditTeam?: any;
+}
 
+const TeamCard = ({ members, teamCode, handleEditTeam }: TeamProps) => {
+    const title = teamCode === "" ? "Team" : "Team " + teamCode;
```

Comment:
is the teamCode empty string or null? It should be null so might need to change how its updated in the slice.
also, get rid of the members and teamCode props, since we're using redux, we should get those values from the slice selectors